### ISSUE TITLE
Testing on different data

### ISSUE THREAD

**Issue by PrithviKambhampati** [sshaoshuai/PointRCNN#109]
Hi, I have a different data set with just the point cloud information. I wanted to know how would I go about to test your model on the data that I have. What are the changes that I need to make for this model to work on that data. I do not have any camera data.

**Comment by bmankirLinker** [comment]
You can follow the code and update in the `eval_rcnn` part and use the cam axis definition Velodyne points to feed your model if you're going to use a KITTI pretrained model for inference. Can use the rounded version of KITTI calibration for V2C if your Velodyne data has the same axis definitions as KITTI Velodyne.

**Comment by DaHaiHuha** [comment]
@PrithviKambhampati My suggestions would be rewrite a  dataloader, if you're still confuesed, maybe you could reference my closed PR and in /dataset/lytf_dataset.py, where i def a new dataloader for only lidar point cloud and labels input

**Comment by yinhai86924** [comment]
> @PrithviKambhampati My suggestions would be rewrite a dataloader, if you're still confuesed, maybe you could reference my closed PR and in /dataset/lytf_dataset.py, where i def a new dataloader for only lidar point cloud and labels input

thank you for your suggestioon, but  can you share your a new dataloader code for only lidar point cloud and labels input
